docs/anomaly-detection/Overview.md

```diff
@@ -35,6 +35,8 @@ In addition to that, setting up alerting rules manually has been proven to be te
 error-prone, while anomaly detection can be easier to set up, and use the same model for different
 metrics.
 
+`vmanomaly` can be used as a helper to set up your own alerting. You can rely on the spikes you see in anomaly scores to form the metric queries for alerting rules.
```

Comment:
@tenmozes  Is it ok to add this use case here?

Reply:
yeap